flash/core/data/data_pipeline.py

```diff
@@ -190,9 +191,11 @@ def _resolve_function_hierarchy(
         elif stage == RunningStage.PREDICTING:
             prefixes += ['predict']
 
+        prefixes += [None]
```

Comment:
This would mean that `default_transforms` has a lower priority as `default_transforms_train`. IMO this will not fix https://github.com/PyTorchLightning/lightning-flash/pull/262#discussion_r633089549. If I'm not mistaken this will pull the default train transform which is unwanted

Reply:
Removed ImageClassificationPrepreprocess, so we don't have this issue.

Reply:
Yes, `train_defaut_transforms` should have a higher priority than `default_transforms`.